ios/FluentUI/Label/Label.swift

```diff
@@ -10,8 +10,17 @@ import UIKit
 /// By default, `adjustsFontForContentSizeCategory` is set to true to automatically update its font when device's content size category changes
 @objc(MSFLabel)
 open class Label: UILabel, TokenizedControlInternal {
-    @objc open var colorStyle: TextColorStyle = .regular {
-        didSet {
+    private static let defaultColorForTheme: (FluentTheme) -> UIColor = TextColorStyle.regular.uiColor
```

Comment:
Note that order of class contents should be `open` -> `public` -> `internal` -> `private`, with `func` coming before `var` or `let` in each group.

https://microsoft.github.io/swift-guide/FileOrganization.html#convention---ordering

Reply:
Fixed with cb00c99